src/scss/_welcome.scss

```diff
@@ -30,6 +30,7 @@
         height: calc(100% - 148px);
```

Comment:
The same is here. I think this `148px` appears from header height, but It should be clearly described, you can do this with comments :-)